doc/source/general/compiling.rst

```diff
@@ -18,7 +18,33 @@ When you try to run your script, the first thing that kOS does is transform your
 
 This process of transforming your script into Machine Language, or "ML" is called "Compiling".
 
-The ":ref:`RUN`" and ":ref:`RUNPATH`" commands do this silently, without telling you. This is why you may have noticed the universe slightly stutter for a moment when you first run your program. Compiling is hard work for the universe to do.
+The :key:`RUN` and :func:`RUNPATH` commands do this silently, without telling
+you.  The :key:`COMPILE` command explicitly compiles the file and saves it for
+future use.
+
+.. note::
+    .. versionchanged:: 1.1.0
+        kOS now performs compilation on a parallel thread (thought to run in a
+        parallel universe).  This means that the universe no longer needs to
+        freeze in order to perform the compilation.  As with the execution of
+        other triggers, compiling from mainline code will not interupt or pause
+        triggers but compiling within a trigger will result in both the mainline
+        and trigger code pausing until the compilation is complete.
```

Comment:
It sounds like you mostly want to make sure we clarify that the universe continues to execute during compilation, instead of essentially being instant in the frozen universe.  Obviously the two of us are going to have our own style/tendency in the way we write the text, so see if this accurately conveys your meaning, without re-writing the whole paragraph (modification in bold):

>kOS now performs compilation on a parallel thread (thought to run in a parallel universe). This means that the universe no longer needs to freeze in order to perform the compilation. **It also means that the process now takes time (possibly several seconds), while the universe continues moving until the compilation is complete.** As with the execution of other triggers, compiling from mainline code will not interrupt or pause triggers but compiling within a trigger will result in both the mainline and trigger code pausing until the compilation is complete.  **There may be many seconds between starting the compilation and executing the code, so you shouldn't assume that the physical properties (mass, position, velocity, etc.) will remain constant during compilation.**

(As I went through and kept adding your thoughts, my revised version got much more wordy than the original one sentence solution I had prepared, it may now be a big enough change that selecting your phrasing or mine might not matter much in the long run)

Given the tone of this document page, I didn't want to break the 4th wall by talking about the underlying system.  I'd be inclined to add your programmer note as a comment in the source instead of in the user docs.  If you think that the underlying implementation is important enough to note though, that's fine.  It probably means we should move this block of text out of a `.. note::` box though and under it's own heading, since it's more than just side note.